ble/Gap.h

```diff
@@ -472,6 +511,126 @@ class Gap {
         *countP = 0; /* Requesting action from porter(s): override this API if this capability is supported. */
     }
 
+    /**
+     * @return Maximum size of the whitelist.
+     */
+    virtual uint8_t getMaxWhitelistSize(void) const
```

Comment:
In this case I was trying to be consistent with the other getMax*() function. See here https://github.com/andresag01/ble/blob/whitelisting/ble/Gap.h#L273.